### PR TITLE
Up 46 add a button to clear cache and clear all data in settings section

### PR DESCRIPTION
**Pull request by Jessemwangi**

### CODE REVIEW

File: src/renderer/App.jsx [hunk 1]
```diff
@@ -125,6 +130,17 @@ const App = () => {
             bookmarksDispatcher={bookmarksDispatcher}
           />
         </Route>
+        {/* <Route exact path="/cleadata">
```
Comment (appketo) [inline src/renderer/App.jsx:133]: Please remove commented out code

File: src/renderer/App.jsx [hunk 2]
```diff
@@ -125,6 +130,17 @@ const App = () => {
...
+          <SiteData
```
Comment (appketo) [inline src/renderer/App.jsx:134]: Also here

File: src/renderer/components/SiteData/SiteData.jsx [hunk 3]
```diff
@@ -0,0 +1,191 @@
...
+
+    return items.filter((item) => {
+      const time = new Date(item.time); //get item.time from the array and asign it to time
+      return time >= cutoff; //filter all the item with date freater than the base to be deleted
+    });
+  }
+
+  return (
+    <div className="container">
+      {/* <>
```
Comment (appketo) [inline src/renderer/components/SiteData/SiteData.jsx:123]: Please remove these commented out code

File: package.json [hunk 4]
```diff
@@ -58,7 +58,9 @@
   },
   "author": "Unelma Platforms Inc.",
   "license": "Apache 2.0",
-  "dependencies": {},
+  "dependencies": {
+    "smalltalk": "^4.1.1"
```
Comment (appketo) [inline package.json:62]: Would it be possible to move this dependency in devDependencies instead

**Review by appketo — COMMENTED** [review]
Added some in-line comments

**Review by appketo — APPROVED** [review]
Much better, and looks clean
Approved

**Review by unellma — APPROVED** [review]
- Pulled requested
- Tested locally
- Changes looks good for me
- Approved

**Pull request merged**